packages/grants-stack-data-client/src/openapi-search-client/.openapi-generator-ignore

```diff
@@ -0,0 +1,23 @@
+# OpenAPI Generator Ignore
```

Comment:
Eventually we might want to gitignore the whole `openapi-search-client` folder and make its generation into part of the build process to prevent people from modifying it by hand, but for now I think we're good